api4/plugin_test.go

```diff
@@ -891,7 +918,10 @@ func TestGetInstalledMarketplacePlugins(t *testing.T) {
 
 		plugins, _, err = th.Client.GetMarketplacePlugins(&model.MarketplacePluginFilter{})
 		require.NoError(t, err)
-		require.Equal(t, expectedPlugins, plugins)
+
+		require.Len(t, plugins, 2)
+		require.Equal(t, "server-and-webapp", plugins[0].Manifest.Id)
+		require.Equal(t, "webapp-only", plugins[1].Manifest.Id)
```

Comment:
This is written this way, because when the `local_only` param is used, an extra label is put on each plugin saying that the plugin is not available in the marketplace. So the structs arent directly equal past the `BaseMarketplacePlugin`